src/browser/modules/D3Visualization/lib/visualization/components/mouseEventHandlers.ts

```diff
@@ -0,0 +1,146 @@
+import { D3DragEvent, drag as d3Drag } from 'd3-drag'
+import { Simulation } from 'd3-force'
+import { BaseType, Selection } from 'd3-selection'
+import { ZoomBehavior } from 'd3-zoom'
+
+import {
+  DEFAULT_ALPHA,
+  DEFAULT_ALPHA_TARGET,
+  DRAGGING_ALPHA_TARGET
+} from '../constants'
+import Relationship from './Relationship'
+import VizNode from './VizNode'
+
+export const zoomEventHandler = (
+  selection: Selection<SVGGElement, unknown, BaseType, unknown>,
+  baseGroup: Selection<SVGGElement, unknown, BaseType, unknown>,
+  zoomBehavior: ZoomBehavior<SVGGElement, unknown>
+) => {
+  const handleZoomOnShiftScroll = (e: WheelEvent) => {
+    if (e.shiftKey) {
+      e.preventDefault()
+
+      const delta =
+        -e.deltaY * (e.deltaMode === 1 ? 0.05 : e.deltaMode ? 1 : 0.002)
```

Comment:
here's some more magic numbers that we haven't moved to constants. I don't fully understand what the numbers mean, so I'm not sure what I would name the constants, so perhaps they should stay as inline numbers for now 🤷

Reply:
I'm not entirely sure what they represent, I copied this from the D3 docs. I'll add a link to it in case we actually need to understand it better later on. Also, I tried using the method that supposedly returns a function that calculates this, but couldn't get it to work (it seems like it's requiring different arguments than the documented ones).

Reply:
Sounds good to add a comment and a link to make it clear that it is the default function we are using. I also tried to use the method that returns this default function and it is actually just typescript that complains, when I added ts-ignore it worked in runtime, so it seems to be a bug in the type definitions of this. Anyway I guess it is not good to have ts-ignore so I think it is better you add a comment and a link for now and we can update it later if the typescript problem is resolved.